content/docs/connectors/oauth.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Authentication Through an Generic OAuth 2.0 Provider"
+linkTitle: "Generic OAuth 2.0"
+description: ""
+date: 2021-03-15
+draft: false
+toc: true
+weight: 2050
+---
+
+## Overview
+
+Dex users can make use of this connector to work with standards-compliant [OAuth 2.0](https://oauth.net/2/) authorization provider, in case of that authorization provider is not in the Dex connectors list.
```

Comment:
```suggestion
Dex users can make use of this connector to work with standards-compliant [OAuth 2.0](https://oauth.net/2/) authorization providers, in case of that authorization provider is not in the Dex connectors list.
```